### ISSUE TITLE
Weapons & Equipment - Racially Unique Items

### ISSUE THREAD

**Issue by Yther** [hammerheim-game/hammerheim-rules#51]
There are a shitload of items in the game that are racially unique. The battle monks were where I first started looking but there's a wealth of others that either one or no warbands have access to. For the sake of avoiding bloating the book I feel like it might be a good idea to have those unique weapons appear in the warband rosters instead. I mean is there any reason Clan Pestilens need to rummage through the book to the equipment section to find out what a Plague Sword does when they're the only faction that can use it?

The equipment section would therefore be a much shorter, more concise, and easy-to-read part of the book that only has the generic items that most people can buy.

**Comment by Yther** [comment]
There are -SO- many items for which this is the case. If its a subject you want to tackle I can provide a comprehensive list.

**Comment by mcclowes** [comment]
J'agree

**Comment by mcclowes** [comment]
I agree with this.
I think just stick them at the end of each warband. That way it won't actually mean rejigging the existing layouts

**Comment by mcclowes** [comment]
I agree, stick them a tthe bottom of the warband section so it doesn't mean rejigging layouts

**Comment by Yther** [comment]
Faction-unique items that don't need to be in the Equipment section.

Ball and Chain
Barbed Spear
Boss Pole
Brazier Iron
Censer
Claw of the Old Ones
Diseased Dagger
Horseman's Hammer
Katar
Plague Sword
Saerath
Serpent Whip
Sigmarite Warhammer
Squig Prodder
Star Blade
Star Sword
Steel Whip
Weeping Blades

Cold Metal Weapons
Blowpipe
Bolas
Bolt Thrower
Repeater Bolt Thrower
Sunstaff
Sun Gauntlet
Tufenk

Hand-held Mortar
Nuln Shotgun
Repeater Handgun
Repeater Pistol
Small Cannon
Warpfire Thrower


Pavise
Sea Dragon Cloak
Clan Pestilens Banner
Standard of Nagarythe

Bear-Claw Necklace
Red Toof Tribal Jewellery

Hammer of Witches
Magic Gugginz

War Horn of Nagarythe

**Comment by mcclowes** [comment]
Cool, will executre this change in this order:
1) Merge in #52
2) Do warband equipment tables
3) Move these item descriptions into their respective warband list

Max

maxcc@me.com (mailto:maxcc@me.com)
+44 (0) 7976 136 097

**Comment by Yther** [comment]
I'm not sure I quite follow :o I'm working on the Warband sheets at the moment and including the unique items into them as I go, there's no need to move the descriptions into the lists or that's going to cause a huge pileup of conflicts. All that needs doing is removing those items from the Price Chart and from the Equipment chapter of the book.

**Comment by Yther** [comment]
Basically, just delete them from the equipment section, they're already in the warbands
